src/_data/codebase/v2_3/commerce/bin-magento.json

```diff
@@ -10718,7 +10718,7 @@
                   "is_value_required" : false,
                   "description" : "The maximum expected execution time of deployment static process (in seconds).",
                   "accept_value" : true,
-                  "default" : 400,
+                  "default" : 900,
```

Comment:
Need to revert the changes to the .json files:

The content in thes files is auto-generated from the Magento codebase.


Any manual changes will be overwritten the next time we release. Additionally, manual changes might not accurately reflect what is in the code base for a release, so the docs will not be accurate.

Reply:
Reverted